src/net/eth.ts

```diff
@@ -290,7 +291,7 @@ export default class Eth extends Api {
       // call ether contract for ether
       if (isEther(asset)) {
         return ethContract.methods
-          .lock(polkadotRecipientBytes, channelId, parachainId)
+          .lock(polkadotRecipientBytes, channelId, parachainId,TRANSACTION_FEES)
```

Comment:
You would have to plumb the data down from the list the same way we do `parachainId`.

Reply:
As suggested, I've updated the changes.